### DISCUSSION TITLE
Set default number of items per page to a different value

### DISCUSSION THREAD

**Discussion by JN8086** [stashapp/stash#3321] [Q&A] [answered]
Is there a way to set the default number of items per page to a different value? Currently it is always 25, and I'd like it to be higher.

Or maybe a way to let it scroll infinitely?

**Comment by DogmaDragon** [comment, marked as answer]
Select the number you want and then click the bookmark icon and `Set as default`. 
![image](https://user-images.githubusercontent.com/103123951/210085964-e4e2af77-04f7-4736-bd84-01570f3dd2fc.png)

**Reply by JN8086** [reply]
thanks!
